src/app/components/FoodCard/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/image';
+
+import FoodImageSvg from '../../../../public/images/food-image.svg';
+import styles from './styles.module.css';
+
+export interface FoodCardProps {
+  title: string;
+  description: string;
+  foodImage: string;
+  price: number;
+  discountedPrice?: number;
+  discountPercentage?: number;
+}
+
+export default function FoodCard({
+  title,
+  description,
+  foodImage,
+  price,
+  discountedPrice,
+  discountPercentage,
+}: FoodCardProps) {
+  return (
+    <article className={styles.foodCard}>
+      <div className={styles.foodImageWrapper}>
+        <Image src={FoodImageSvg} alt="Foto da comida" fill />
+      </div>
+      <div className={styles.foodCardTextContent}>
+        <div className={styles.foodCardTextWrapper}>
+          <h3 className={styles.foodCardTitle}>{title}</h3>
+          <p className={styles.foodCardDescription}>{description}</p>
+        </div>
+        <p className={styles.foodCardPricesWrapper}>
+          {discountedPrice && (
+            <span className={`${styles.foodCardDiscountPrice}`}>
+              R$ {discountedPrice}
+            </span>
+          )}
```

Comment:
```suggestion
          {discountedPrice ? (
            <span className={`${styles.foodCardDiscountPrice}`}>
              R$ {discountedPrice}
            </span>
          ) : null}
```

To prevent avoidable UI bugs, use the JavaScript ternary operator for conditional rendering of React components instead of the logical AND operator. It’s a simple spell but quite unbreakable [click here to read more...](https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8)